lib/src/query_component.html

```diff
@@ -0,0 +1,17 @@
+<input [(ngModel)]="advertiserId"
+       placeholder="Advertiser ID" debugId="advertiser-id-input">
+<input [(ngModel)]="mediaPlanId"
+       placeholder="Media Plan ID" debugId="media-plan-id-input">
+<input [(ngModel)]="insertionOrderId"
+       placeholder="Insertion Order ID" debugId="io-id-input"><br>
+<div *ngFor="let choice of queryTypeChoices">
+    <input type="radio" (click)="queryType = choice" name="radio-group"
+           id="{{enum2String(choice)}}-radio-btn"/>
```

Comment:
right now the user will be seeing camelCase var name as the button label. Is there a way to display a user-friendly string here?